changes/713.feature.md

```diff
@@ -0,0 +1 @@
+Introduce vfolder status by adding enumtype column to control user access to individual vfolders
```

Comment:
just a little comment.
it is good to write only **what** changes not **how** in news fragment.
it would be better to remove "by adding enumtype column".